web-marketplace/src/pages/BuyCredits/BuyCredits.tsx

```diff
@@ -39,7 +39,7 @@ export const BuyCredits = () => {
   } = useGetProject();
 
   const [paymentOption, setPaymentOption] = useState<PaymentOptionsType>(
-    PAYMENT_OPTIONS.CRYPTO,
+    PAYMENT_OPTIONS.CARD,
```

Comment:
why this change? this results in the crypto option not being selected by default if there's no fiat sell orders:

![image](https://github.com/user-attachments/assets/a81819ba-1c1b-4559-9297-6a72e34952d6)

the card option is greyed out but we still see the related form and the user needs to click on "buy with crypto" which feels a bit weird

Reply:
The reason behind this change was that the user balance validation was getting triggered in the fiat form because the payment method was crypto.. but I'm unable to reproduce this bug anymore so reverting back to card.

Although, arguably, the default option should be card, and if there is no `cardSellOrders` we should update the `paymentOption` to crypto  in a `useEffect`, wdyt?

Reply:
it's done the other way around right now, and switching to card in a use effect if there are some cardSellOrders 
I would prefer to keep it this way just to avoid any extra work